db/seeds.rb

```diff
@@ -5,3 +5,27 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create!(
+  email: 'janewang0@yahoo.com',
+  password: 'password'
+)
+
+10.times do
+  User.create!(
+    email:    Faker::Internet.email,
+    password: Faker::Internet.password(8),
+  )
+end
+users = User.all
+
+30.times do
+  Item.create!(
+    name: Faker::Lorem.sentence
+  )
```

Comment:
These items are not tied to users. Please make sure that when you seed your database, each User gets Items. Check Bloccit or Blocipedia for an example.